docs/guides/universal-profile/create-profile.md

```diff
@@ -308,22 +308,22 @@ async function createUniversalProfile() {
 
 createUniversalProfile();
 
-// my Universal Profile address: 0xEde1198b046d8ED64B71adeA5d3B7370cc84A7FB
+// my Universal Profile address: 0xbd9Ef16132258ACbfEd899F594C45772604C6509
 ```
 
-We can also visualize our UP on the [l16.universalprofile.cloud](https://l16.universalprofile.cloud) website by adding the address of the deployed UP in the URL, after the `/` (slash), as follow:
+We can also visualize our UP on the [universalprofile.cloud](https://universalprofile.cloud) website by adding the address of the deployed UP in the URL, after the `/` (slash), as follow:
```

Comment:
As of right now, this is not correct.

Reply:
replaced it with wallet.universalprofile.cloud and enforced smart contract version